tests/write_font_test.py

```diff
@@ -222,15 +226,15 @@ def test_vertical_metrics(ascender, descender, linegap):
         (
             ("reused_shape_2.svg",),
             "reused_shape_2_picosvg.ttx",
-            {"color_format": "picosvg"},
+            {"color_format": "picosvg", "pretty_print": True},
         ),
         # Safari can't deal with gradientTransform where matrix.inverse() == self,
         # we work around it by nudging one matrix component by an invisible amount
         # https://github.com/googlefonts/nanoemoji/issues/268
         (
             ("involutory_matrix.svg",),
             "involutory_matrix_picosvg.ttx",
-            {"color_format": "picosvg"},
+            {"color_format": "picosvg", "pretty_print": True},
         ),
         # Check that we do _not_ make composite glyphs with reused paths if
         # the latter overlap and the transform for the shape reuse is such
```

Comment:
since we are no longer creating glyf composites (with potentially overlapping components that may create winding issues thus is safer to decompose) but reusing PaintGlyphs via PaintTransforms, the comment above about transformed_components_overlap.ttx may be outdated and the whole test probably no longer relevant?
Actually i'd be curious to know whether reusing "components" as PaintTransform-ed PaintGlyphs (as opposed to traditional glyf composites) incur in the same overlap issues (I suspect not, as each coloured layer is composited over the previous one)..